packages/camera/camera_android/CHANGELOG.md

```diff
@@ -28,6 +29,7 @@
 ## 0.10.6+1
 
 * Adds a namespace for compatibility with AGP 8.0.
+* Bump: `com.android.tools.build:gradle:7.2.0`
```

Comment:
This should be a part of the latest changelog entry.

Reply:
Moved to last entry.